lib/ClangImporter/ImportDecl.cpp

```diff
@@ -1792,6 +1792,26 @@ applyPropertyOwnership(VarDecl *prop,
 }
 
 namespace {
+  /// Customized llvm::DenseMapInfo for storing borrowed APSInts.
```

Comment:
It's probably not even necessary, but it's not worse than what was there before, and it's boxed up pretty neatly.